### PR TITLE
Splash and loading text styling

### PR DESCRIPTION
**Pull request by ddelpiano**
This address the issue #193

### CODE REVIEW

**Comment by ddelpiano** [comment]
@Robbie1977 FYI.

File: js/pages/geppetto/geppetto.ejs [hunk 1]
```diff
@@ -45,9 +45,26 @@ $content
 
 
 <div id="mainContainer">
-	<div id="loadingText" style="font-family: consolas;margin: 0px;">
-		<img src="geppetto/build/splash.png" style="max-width: 90%;margin: 0px;position: absolute;display: inline-block;text-align: center;vertical-align: middle;top: 2%;left: 5%;" alt="" onerror='document.getElementById("loadingMessage").style.display="block"' >
-		<span id="loadingMessage" style="display : none; margin-top: 50px" >LOADING... [~3MB]</span>
+	<div id="loadingText">
+		<div style="position: absolute;
```
Comment (filippomc) [inline js/pages/geppetto/geppetto.ejs]: @dario I suggest to use flex to achieve the centering. Fare easier and robust. https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container.

Comment (ddelpiano) [reply js/pages/geppetto/geppetto.ejs]: @filippomc I tried and it did not work well with all the splash I tested, this is the solution that so far works well with all the images tried.

Comment (filippomc) [reply js/pages/geppetto/geppetto.ejs]: image
 I see this with netpyne. Do we actually want to hide the text when the splash is available? I can try the fix with flex as @gidili suggests

**Comment by ddelpiano** [comment]
pushed changes as requested

**Comment by Dario** [comment]
Just wanted to advice that Dario D. may not be getting notifications of your mentions since I'm @dario :)

**Pull request merged**